src/exts/ping.py

```diff
@@ -0,0 +1,26 @@
+from loguru import logger
+from disnake import Embed
+from nextcord.ext import commands
+
+
+class Ping(commands.Cog):
+    "Send an embed about the bot's ping."
+
+    def __init__(self, bot: commands.Bot):
+        self.bot = bot
+
+    @commands.command(name="ping")
+    async def ping(self, ctx: commands.Context) -> None:
+        """Ping the bot to see its latency and state."""
+        logger.debug(f"Command `{ctx.invoked_with}` used by {ctx.author}.")
+        embed = Embed(
+            title=":ping_pong: Pong!",
+            description=f"Gateway Latency: {round(self.bot.latency * 1000)}ms",
+        )
+
+        await ctx.send(embed=embed)
+
+
+def setup(bot: commands.Bot) -> None:
+    """Load the Ping cog."""
+    bot.add_cog(Ping(bot))
```

Comment:
Do we want this to be an extension that can accidentally be unloaded? I don't remember how Lancebot or Python does it. I think the load/unload commands are in an extension so maybe.

I think it is something we should think twice about though.

Reply:
I have it loaded directly in the `__main__` currently, I think we would eventually use the extension loading like Sir-Lancebot when we have more commands / extensions.